internal/stage13.go

```diff
@@ -21,6 +21,7 @@ func testNumbers(stageHarness *test_case_harness.TestCaseHarness) error {
 		TestCases: []testcases.TokenizeTestCase{
 			{FileContents: getRandIntAsString(), ExpectsError: false},
 			{FileContents: fmt.Sprintf("%d.%d", getRandInt(), getRandInt()), ExpectsError: false},
+			{FileContents: fmt.Sprintf("%d.0000", getRandInt()), ExpectsError: false},
 			{FileContents: shuffledString1, ExpectsError: false},
```

Comment:
@andy1li let's remove the first shuffledString case then, and only keep the second (we can rename it to `shuffledString1`. - we've been sticking to a pattern of having exactly 4 test cases as much as possible